README.md

```diff
@@ -31,6 +31,14 @@ Default: `true`
 
 Type: `bool`
 
+### sudo_check_if_configured
+
+This variable provides semantic check of already configured sudoers in case ansible setup is not needed and it is skipped.
```

Comment:
I do not understand in which case users might need to set it to `true`/`false`

Reply:
Is it a way to ensure idempotency? If it is, do we really want users to be able to set it to `false`?

Reply:
you got it right. if it is set to false, sudoers will be rewritten [forcely]